projects/discussion-ui/src/lib/components/discuss-start/discuss-start.component.ts

```diff
@@ -84,7 +84,7 @@ export class DiscussStartComponent implements OnInit {
   isFieldValid(field) {
     const valueNoWhiteSpace = this.startForm.value[field];
     if (valueNoWhiteSpace) {
-      if (valueNoWhiteSpace.trim() === '') {
+      if (valueNoWhiteSpace.trim() === '' || valueNoWhiteSpace !== valueNoWhiteSpace.trim()) {
```

Comment:
Why are we allowing user to type empty space for the first charecter. Disable/ignore is user is giving space at the first charecter itself.